processing/src/test/java/io/druid/query/aggregation/HistogramAggregatorTest.java

```diff
@@ -108,14 +107,10 @@ public void testBufferAggregate() throws Exception
 
     final TestFloatColumnSelector selector = new TestFloatColumnSelector(values);
 
-    ArrayList<Float> b = Lists.newArrayList();
-    for (int i = 0; i < breaks.length; ++i) {
-      b.add(breaks[i]);
-    }
     HistogramAggregatorFactory factory = new HistogramAggregatorFactory(
         "billy",
         "billy",
-        b
+        Floats.asList(breaks)
```

Comment:
This is not the same functionality as the prior. The prior created an independent list instead of having the `breaks` be the backing array. is that intended?